Recipe/recipe.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title> My Recipe Page</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Sarah's Recipe Page</title>
+  <link href="./recipe.css" rel="stylesheet" type="text/css" />
+  
+
+
+  <head>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Teko|Abril Fatface|Yeseva One|Oleo Script|Tangerine|Assistant">
+    <style>
+    h1 {font-family: "Oleo Script", serif; font-size: 62px; text-shadow: 4px 4px 4px rgb(124, 152, 123); color: rgb(34, 77, 31)}
+    h2 {font-family: "Assistant", serif; font-size: 25px; color: rgb(0, 0, 0)}
+    </style>
+    </head>
+
+
+  
+</head>
+
+<body>
+  
+ <h1 class="center" style="margin-bottom: 0"><b>Imbaattan Recipe</b></h1> 
+ <h2 class="center"> (Fried Potato Wedges with Minced Meat) </h2>
+  
+
+  
+<div class="recipe-image" id="specialdiv">
+<img src="https://arousingappetites.com/wp-content/uploads/2022/08/MBattan.jpeg?ezimgfmt=ng%3Awebp%2Fngcb1%2Frs%3Adevice%2Frscb1-1" width="250" height="200" class="center" style="margin-bottom: 50px">
+</div>
+<p class="center"><b>Description:</b><br>This delicious deep fried finger food originates and is unique to Libya,<br> my home country in North Africa! <br> It's especially popular during special occasions as well as Ramadan.<br> The well seasoned flavours of the meat and the combination <br> of the crispy exterior of the potatoes might <br> make this your new favourite side dish! <br> North African recipes are passed down generationally and typically <br> don't have exact measurments. <br> Below is the closest appoximation of measurments <br> to the recipe my mother makes.<p class="center">Enjoy and Sahtain! (Arabic word for Bon Appetit!) </p>   </p>
```

Comment:
It's not valid for you to have a `<p>` inside of a `<p>`. Either have them as separate paragraphs (i.e. being siblings), OR just not have the nested one (if you need to add a class to the last one, use a `<span>` instead. `<span>` tags are valid inside of `<p>` as well as inside other `<span>` tags).

Also... I'd recommend writing with newlines after each `<br>`. Word wrap exists but........ It's not super common for devs to use it because it often leads to weird wrapping that gets confused with indentation.